docs/UserGuide.md

```diff
@@ -630,4 +582,14 @@ You may copy and paste multiple lines of commands into the command box. Press th
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous PropertyWhiz home folder.
+**A**: Simply install the app on the other computer and overwrite the empty data file it creates with the data file located at your previous PropertyWhiz home folder.
+
+
+## Glossary
+
+Word | Meaning
+-----|------------------
+Command Line Interface (CLI) | Text-based user interface
+Graphical User Interface (GUI) | User interface that allows users to interact via mouse and graphics
+Alphanumeric | Containing letters and numbers only
+CSV file | Comma-separated values file, a file format Microsoft Excel can export to
```

Comment:
can also add 
` JAR file | Java ARchive file, an application file format for Java programs` or something like that

Reply:
As mentioned above, JSON can be defined here.